source/reference/api/start.txt

```diff
@@ -151,12 +225,104 @@ Request
 ~~~~~~~~
 
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Example 3 - Start Sync from Replica Set to Sharded Cluster
+----------------------------------------------------------
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/start-rs-shard.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
 Behavior
 --------
 
+State
+~~~~~
+
 If the ``start`` request is successful, ``mongosync`` enters the
 ``RUNNING`` state.
+
+Shard Collections
+~~~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option. This option configures how ``mongosync`` shards
+collections.
+
+The ``sharding.shardEntries`` array specifies the collections to shard.
+Collections that are not listed in this array replicate as unsharded.
+
+Supporting Indexes
+~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster can automatically
+create a supporting index for the new shard key on the destination cluster.
+This is done by setting the ``sharding.createSupportingIndexes`` option.
+
+When ``sharding.createSupportingIndexes`` is ``false`` (the default): 
+
+* Each shard key you provide for the ``sharding.shardEntries`` option
+  must have an existing index on the source cluster.
+
+* One of the indexes used for the shard key must have simple collation if the 
+  collection uses any other collation.
+
+* To use a unique index in the shard key, you must specify its uniqueness when
+  you create the index on the source cluster. 
+
+* Unique indexes on the source cluster that are incompatible with the requested
+  shard key on the destination cluster, such as a unique index on the source
+  that does not contain the requested shard key as a prefix on the destination,
+  can cause ``mongosync`` to fail.
+
+When ``sharding.createSupportingIndexes`` is ``true``: 
+
+* ``mongosync`` synchronizes indexes on the source cluster to the destination.
+
+* If the supporting indexes exist on the source cluster, ``mongosync`` 
+  synchronizes the indexes to the destination cluster and uses them 
+  as the shard key.
+
+* If the supporting indexes don't exist, ``mongosync`` creates them on the
+  destination cluster.
+
+The ``sharding.createSupportingIndexes`` option affects all collections sharded
+during sync.
+
+
+Rename During Sync
+~~~~~~~~~~~~~~~~~~
+
+Collections listed in the ``sharding.shardEntries`` array 
+when synchronized from the replica set to a sharded cluster 
+become sharded collections on the destination cluster.
+
+Renaming a collection (such as with the :dbcommand:`renameCollection` command)
+on the source cluster after calling ``start`` but before ``mongosync`` begins 
+to copy the collection can block the collection from sharding on the destination.
+
+To check whether a collection is safe to rename, call the :ref:`c2c-api-progress`
```

Comment:
Technically this is not per collection: progress will return > 0 only once it is safe for **all collections**.

Reply:
Fixed